test/parallel/test-node-run.js

```diff
@@ -251,12 +251,12 @@ describe('node --run [command]', () => {
     assert.strictEqual(child.code, 1);
   });
 
-  it('returns error when --run-in is used without --run', async () => {
+  it('returns error when --run-dir is used without --run', async () => {
```

Comment:
I thought this place should use run --dir @aduh95

Reply:
this was for misuse, thank you. I edited it this way:

https://github.com/nodejs/node/pull/57523/commits/bd6f91087936f25ec12ac7185af3d81e73ca3f20